pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -537,8 +551,8 @@ func (lbaas *LbaasV2) UpdateLoadBalancer(clusterName string, service *api.Servic
 
 	// Compose Set of member (addresses) that _should_ exist
 	addrs := map[string]empty{}
-	for _, nodeName := range nodeNames {
-		addr, err := getAddressByName(lbaas.compute, types.NodeName(nodeName))
+	for _, node := range nodes {
+		addr, err := nodeAddressForLB(node)
```

Comment:
I don't think so?  I'm given the list of nodes as an argument, and I want to configure the LB with the whole lot and have the LB healthchecks react to "dynamic" readyness.
(If the `UpdateLoadBalancer` API is intended to mean otherwise, please let me know)